src/include/btree_inline.h

```diff
@@ -255,6 +255,24 @@ __wt_btree_bytes_updates(WT_SESSION_IMPL *session)
     return (__wt_cache_bytes_plus_overhead(cache, __wt_atomic_load64(&btree->bytes_updates)));
 }
 
+/*
+ * __wt_btree_shared --
+ *     Given a tree's URI and config, determine whether it's shared.
+ */
+static WT_INLINE int
+__wt_btree_shared(WT_SESSION_IMPL *session, const char *uri, const char **bt_cfg, bool *shared)
```

Comment:
There is a flag in the block manager "is_remote". Can we use that instead?

Reply:
I don't think so, we sometimes need to call this before `blkcache_open` (e.g. in the import case, where it mustn't get as far as opening if it's shared).